doorstop/core/tree.py

```diff
@@ -176,9 +176,13 @@ def _place(self, document):
             log.info("parent options: {}".format(prefixes))
             raise DoorstopError(msg)
 
+        self.regenerate_children()
+
+    def regenerate_children(self):
```

Comment:
What was the warning that triggered this change? I wonder if this method should be kept private for now (`_regenerate_children()`).